src/pages/HomePage/HomePage.jsx

```diff
@@ -13,7 +15,7 @@ const HomePage = () => {
           <AddNote />
           <div className='inner-container'>
             <div className='note-list'>
-              {notes.map((note) => {
+              {filterByDate.map((note) => {
```

Comment:
You could remove the curly braces and return keyword

Reply:
Done thanks for ponting out